.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.31.0  # MSRV
+          - 1.34.0  # MSRV
```

Comment:
1.32 had an issue with extern crates and `backtrace` (I think this is a dep of `failure`) and I'm now using `try_into` which is stabilized as of 1.34.

Reply:
Yeah, having a recent MSRV is fine.